arm9/lib/include/NNS_g3d.h

```diff
@@ -0,0 +1,89 @@
+#ifndef GUARD_NNS_G3D_H
+#define GUARD_NNS_G3D_H
+
+#include "fx.h"
+
+typedef enum
+{
+    NNS_G3D_GLB_FLAG_FLUSH_WVP              = 0x00000001,
+    NNS_G3D_GLB_FLAG_FLUSH_VP               = 0x00000002,
+    NNS_G3D_GLB_FLAG_INVBASE_UPTODATE       = 0x00000004,
+    NNS_G3D_GLB_FLAG_INVCAMERA_UPTODATE     = 0x00000008,
+    NNS_G3D_GLB_FLAG_INVPROJ_UPTODATE       = 0x00000010,
+    NNS_G3D_GLB_FLAG_INVBASECAMERA_UPTODATE = 0x00000020,
+    NNS_G3D_GLB_FLAG_INVCAMERAPROJ_UPTODATE = 0x00000040,
+
+    NNS_G3D_GLB_FLAG_BASECAMERA_UPTODATE    = 0x00000080,
+
+    NNS_G3D_GLB_FLAG_SRTCAMERA_UPTODATE     = NNS_G3D_GLB_FLAG_BASECAMERA_UPTODATE,
+    NNS_G3D_GLB_FLAG_FLUSH_ALT              = NNS_G3D_GLB_FLAG_FLUSH_WVP
+}
+    NNSG3dGlbFlag;
+
+typedef struct
+{
+    /*0x000*/ u32     cmd0;
+    /*0x004*/ u32     mtxmode_proj;
+    /*0x008*/ MtxFx44 projMtx;
+
+    /*0x048*/ u32     mtxmode_posvec;
+    /*0x04C*/ MtxFx43 cameraMtx;
+
+    /*0x07C*/ u32     cmd1;

```

Comment:
can you be a bit more specific what cmd0/cmd1 etc is? it's kinda vague, if not by variable names then maybe a comment?